lighthouse-cli/bin.ts

```diff
@@ -269,8 +273,15 @@ function saveResults(results: Results,
       promise = promise.then(_ => assetSaver.saveAssets(artifacts, results.audits, resolvedPath));
     }
 
-    if (flags.output === Printer.OutputMode[Printer.OutputMode.pretty]) {
-      promise = promise.then(_ => Printer.write(results, 'html', `${resolvedPath}.report.html`));
+    if (flags.output === Printer.OutputMode[Printer.OutputMode.none]) {
```

Comment:
Unfortunate that one cannot get both pretty output and HTML output now, but we're killing it entirely and #1714 captures multiple output desires anyway